tpgtools/overrides/cloudbuildv2/samples/connection/ghe.tf.tmpl

```diff
@@ -0,0 +1,70 @@
+resource "google_secret_manager_secret" "private-key-secret" {
```

Comment:
In the existing examples, two spaces instead of tabes are used for indentations. Can you please change the indentations to two spaces? Thanks.